src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestUpdateCliGit.java

```diff
@@ -77,7 +77,7 @@ public void testSubmoduleUpdateWithThreads() throws Exception {
                 .execute();
         w.git.submoduleInit();
         w.git.submoduleUpdate().threads(3).execute();
-
+        Thread.sleep(3000);
```

Comment:
given `execute()` does not return a Future and `GitCommands` can take a timeout, shouldn't this be method be considered synchronous?
As such isn't this a production issue not a test issue?

Reply:
I'm not willing at this point to spend the time to investigate if the method is synchronous.  I had never seen this test fail in my tests on ci.jenkins.io or in my home lab, but it was a harmless addition, so I accepted it.  Will propose a revert to tidy the test a little.

Reply:
Submitted as:

* #1151